core/src/com/unciv/MainMenuScreen.kt

```diff
@@ -230,9 +231,15 @@ class MainMenuScreen: BaseScreen() {
     }
 
     private fun openCivilopedia() {
-        val ruleset =RulesetCache[game.settings.lastGameSetup?.gameParameters?.baseRuleset]
-            ?: RulesetCache[BaseRuleset.Civ_V_GnK.fullName]
-            ?: return
+        var ruleset: Ruleset? = RulesetCache[BaseRuleset.Civ_V_GnK.fullName]
+        val rulesetParameters = game.settings.lastGameSetup?.gameParameters
+        if (rulesetParameters != null) {
+            ruleset = RulesetCache.getComplexRuleset(rulesetParameters)
+            UncivGame.Current.translations.translationActiveMods = LinkedHashSet(rulesetParameters.getModsAndBaseRuleset())
+        }
```

Comment:
Do we need to reassign `ruleset` if it has been obtained on the very first line?

Reply:
Good point.